testsuites/syncgateway/functional/tests/test_xattrs.py

```diff
@@ -90,11 +91,15 @@ def test_olddoc_nil(params_from_base_test_setup, sg_conf_name):
     # Create clients
     sg_client = MobileRestClient()
     cbs_ip = host_for_url(cbs_url)
-    if cluster.ipv6:
-        sdk_client = Bucket('couchbase://{}/{}?ipv6=allow'.format(cbs_ip, bucket_name), password='password', timeout=SDK_TIMEOUT)
+    if ssl_enabled and cluster.ipv6:
```

Comment:
is timeout not needed?

Reply:
added back. Just wondering why SDK_TIMEOUT added only to this test?